templates/primary-secondary/primary-secondary.js

```diff
@@ -921,12 +921,13 @@ class TemplatePrimarySecondary extends FocusVisiblePolyfillMixin(RtlMixin(Locali
 
 	render() {
 		let tabindex;
+		const size = this._size ?? 0;
```

Comment:
Can we just init `this._size = 0;` in the constructor?

Reply:
Haha see my earlier comments...

Reply:
Not unless we change the [logic here](https://github.com/BrightspaceUI/core/pull/3084/files#diff-06cfe552d3f9e7117999c057f42fd8b028a379c2a88a4004b6cc8ab40b63285fR1044) too, which expects `this._size` to be undefined to trigger its initial calculations.

Reply:
If we prefer that route I can do that, and just add an extra boolean flag like `this._sizeIsInitialized = false` - do we like that better?

Reply:
I'm good either way.